### ISSUE TITLE
Document nested virtualization inside of Qubes (nested VMs in Qubes, not Qubes in a VM)

### ISSUE THREAD

**Issue by johnandrewny** [QubesOS/qubes-issues#2887]
### General notes:
A lot of people needs nested vm, there is some links:

https://www.reddit.com/r/Qubes/comments/5tbe3e/nested_virtualization/  

https://serverfault.com/questions/858039/when-creating-vm-with-virt-manager-on-xen-under-debian-9-stretch-receive-the-err

---

**Comment by schnurentwickler** [comment]
@johnandrewny Why don't you begin with collecting text (and their links) from qubes mailing lists about that topic.
Then other helpful people can write a documentation about. =]

**Comment by TBK** [comment]
Here is a generic source about Xen VM nesting - https://wiki.xenproject.org/wiki/Nested_Virtualization_in_Xen

**Comment by marmarek** [comment]
Related: #4104, including info how to enable it. But as can be seen there, it's broken (enabling it cause instant system crash).

**Comment by lucaszanella** [comment]
I'd like to use it to run Android Studio with emulator.

Any updates on how to enable it without crashing the system?

**Comment by andrewdavidwong** [comment]
> I'd like to use it to run Android Studio with emulator.
> 
> Any updates on how to enable it without crashing the system?

I think this was discussed on the mailing lists. Please try searching the archives or asking there:

https://www.qubes-os.org/support/

**Comment by ben-grande** [comment]
Was documented:

https://doc.qubes-os.org/en/latest/introduction/faq.html#can-i-install-qubes-in-a-virtual-machine-e-g-on-vmware

Possibly purposefully short, to not incite users into trying it.

**Comment by marmarek** [comment]
No, this ticket is about documenting nested virtualization within qubes - like running virtualbox inside a qube (documenting as not supported and not working), not running whole qubes os nested inside something else (kvm/virtualbox etc).
